tests/zerocashTest.cpp

```diff
@@ -612,4 +664,9 @@ int main(int argc, char **argv)
     std::cout << "\nMintTxTest result => " << mintTxResult << std::endl;
     std::cout << "\nPourTxTest result => " << pourTxResult << std::endl;
     std::cout << "\nSimpleTxTest result => " << simpleTxResult << std::endl;
+    std::cout << "\nSaveAndLoadKeysFromFiles result => " << saveAndLoadKeysFromFiles << std::endl;
+
+    return !(merkleSimpleResult && addressResult &&
+        coinResult && mintTxResult && saveAndLoadKeysFromFiles
+        && pourTxResult && simpleTxResult);
```

Comment:
[does not block] Another followup: replace this ``main()`` with a test framework.